test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -258,8 +258,8 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
     }
 
     $testCase = @(
-        @{ proxy_address = "http://localhost:8080"; name = 'http_proxy'; protocol = 'http' }
-        @{ proxy_address = "http://localhost:8080"; name = 'https_proxy'; protocol = 'https' }
+        @{ proxy_address = "http://localhost:38080"; name = 'http_proxy'; protocol = 'http' }
```

Comment:
I think you can use port 9 which is reserved for the Discard Protocol.  So even if the system is listening, it's supposed to discard that TCP request (UDP may use it for Wake-on-Lan, but doesn't affect this).